studio.config.json

```diff
@@ -16,6 +16,13 @@
           "FeedbackButton"
         ]
       ],
+      [
+        "icons",
+        [
+          "icon",
+          "custom-icon"
```

Comment:
is custom icon the right name? (Same comment for `CustomIcon.tsx`)

Reply:
Perhaps AccessibleIcon? The real value we are adding is just wrapping an SVG in the Radix-ui accessibility component.